examples/StorageAccount/StorageAccount.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+BeforeAll {
+  Import-Module Az-InfrastructureTest
+}
+
+Describe 'Verify Storage Account Exists' {
+    it 'Should contain a Storage Account with the given name' {
+        #arrange
+        $rgName = 'rg-test'
+        $serverName = 'azbenchpressstorage'
+
+        #act
+        $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $serverName
```

Comment:
Change $serverName names in this file to $name or $stgAcctName

Reply:
good catch! thanks 👍